docs/developer/DEVELOP.md

```diff
@@ -52,9 +52,9 @@ RH_CERT_URL="https://foo.bar"
 # otherwise all the writes are performed only locally in OSIDB
 BBSYNC_SYNC_TO_BZ=1
 
-# Enable https proxy (optional)
-# This variable is only required when using taskman to access Red Hat's stage environment
-HTTPS_PROXY="http://foo.bar"
+# Enable https proxy in taskman test cases (optional)
+# This variable is only used in case you access a Jira stage environment behind a proxy
+HTTPS_TASKMAN_PROXY="http://foo.bar"
```

Comment:
This should say something like:

"Run taskman tests behind a proxy (optional)
 This variable is only necessary if rewriting the taskman cassettes locally and using the Stage Red Hat JIRA instance, which requires a proxy to be accessed."